app/ui/src/components/Register.vue

```diff
@@ -84,35 +88,54 @@ export default {
       username: '',
       password: '',
       passwordConfirm: '',
-      email: ''
+      email: '',
+      agreeWithTermsAndConfitions: false
```

Comment:
Spelling error here. agreeWithTermsAndCon *f* itios. Shouldn't this be `agreeWithTermsAndConditions`?